lib/fluent/plugin/out_s3.rb

```diff
@@ -97,6 +97,8 @@ def initialize
     config_param :enable_dual_stack, :bool, default: false
     desc "If false, the certificate of endpoint will not be verified"
     config_param :ssl_verify_peer, :bool, :default => true
+    desc "Full path to the SSL certificate authority bundle file that should be used when verifying peer certificates. If unspecified, defaults to the system CA if available."
+    config_param :ssl_ca_bundle, :string, :default => nil
```

Comment:
It would be better that adding also `ssl_ca_directory`.

ref: https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/S3/Client.html#initialize-instance_method

Reply:
Sorry not 100% clear what your asking for, but it sounds like you think we should add both ssl_ca_bundle and ssl_ca_directory? I'm fine with doing both.

Reply:
Okay, I added `ssl_ca_directory` configurations as well.

Reply:
Thanks!